spec/serializers/project_serializer_spec.rb

```diff
@@ -34,7 +34,7 @@
     end
 
     describe 'can filter by state' do
-      let(:paused_project) { create(:full_project, state: "paused", live: false) }
+      let(:paused_project) { create(:full_project, state: "paused", live: true) }
```

Comment:
you may want to add a failing test case to ensure the scope re-assignment doesn't pass these tests.